bones/selectmulti.py

```diff
@@ -13,16 +13,18 @@ def render(self, data, field):
 			result = list()
 
 			for fieldKey in data[field]:
-				if not fieldKey in self.skelStructure[field]["values"].keys():
+				options = {k: v for k, v in self.skelStructure[field]["values"]}
```

Comment:
This variable is evaluated to the same result for every loop entry.
Please put a few lines above, in front of the for-loop.